client/src/screens/events/Detail.js

```diff
@@ -99,7 +142,7 @@ class Detail extends Component {
     this.props.navigation.setParams({
       handleThis: this.mapZoomMe,
     });
-    this.refetchTimer = setInterval(this.onRefresh, 5000);
+    // this.refetchTimer = setInterval(this.onRefresh, 5000);
```

Comment:
i'd rather you remove all code relating to this in one go rather than commenting it. Separate commit would be even more amazing but not necessary.